themes/default/content/enterprise/whitepaper.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Accelerating Enterprise Innovation through Cloud Engineering
+description: |
+    Pulumi is a cloud engineering platform that enables enterprises to use an unified software engineering process to deliver infrastructure and applications together for greater innovation velocity. 
+meta_desc: |
+    Pulumi is a cloud engineering platform that enables enterprises to use an unified software engineering process to deliver infrastructure and applications together for greater innovation velocity. 
```

Comment:
```suggestion
    Pulumi is a cloud engineering platform that gives enterprises a unified process for delivering infrastructure and applications for greater innovation velocity.
meta_desc: |
    Pulumi is a cloud engineering platform that gives enterprises a unified process for delivering infrastructure and applications for greater innovation velocity.
```

Reply:
159! 🤞

Reply:
Ah @susanev looks like you committed just before I suggested.

Reply:
oh i was just trying to get @aaronkao a preview, we can use urs!